apps/ledger-live-mobile/src/screens/ReceiveFunds/03-Confirmation.js

```diff
@@ -272,7 +272,10 @@ export default function ReceiveConfirmation({ navigation, route }: Props) {
               <Trans
                 i18nKey="transfer.receive.verifyPending"
                 values={{
-                  currencyName: mainAccount.currency.managerAppName,
+                  currencyName:
+                    mainAccount.currency.name === "Osmosis"
```

Comment:
We can't have a specific case for osmosis here as this is a generic file.
Using name could be fine but we need to assess that solution. I'll get back to you on this.

Reply:
Yep, this is why we left that comment. We wanted to handle Osmosis but without breaking anything.

Reply:
So after verification using `name` instead of `managerAppName` for all currencies looks perfectly fine :+1:
Can you please just do that change, as well as on the other case L262 a little above? Thanks.

Reply:
Done! https://github.com/LedgerHQ/ledger-live/pull/890/commits/c652baa63b2ebfc1c78b23c5bd1f9d0bec2cdbf7